_includes/docs_menu.html

```diff
@@ -9,7 +9,7 @@
             <a href="{{ site.baseurl }}/docs/trex_overview.html">What is a Dashboard Extension</a>
         </li>
         <li>
-            <a href="{{ site.baseurl }}/docs/ux_design.html" target="_blank">Design Guidelines for Dashboard Extensions</a>
+            <a href="{{ site.baseurl }}/docs/ux_design.html">Design Guidelines for Dashboard Extensions</a>
```

Comment:
@bcantoni  - I put these in here so that the Design Guidelines (a separate guide with its own TOC) would popup in a new tab. That was the idea for the Design Guide and for the Dashboard Extensions API reference, which also has its own TOC. Can re-visit if it's not working.

Reply:
Gotcha - I'll return them.

Reply:
fixed